apps/prairielearn/src/ee/pages/administratorInstitutionSaml/administratorInstitutionSaml.html.ts

```diff
@@ -406,13 +406,11 @@ function DeleteSamlConfigurationModal({ csrfToken }: { csrfToken: string }) {
       </p>
     `,
     footer: html`
+      <input type="hidden" name="__csrf_token" value="${csrfToken}" />
       <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
-      <form method="POST">
```

Comment:
This was actually a mistake. The modal component already creates a form, so this form was invalid since a [form cannot contain another form](https://www.w3.org/TR/xhtml1/#prohibitions). That said, I can't test this.